src/Language/PureScript/TypeChecker/Kinds.hs

```diff
@@ -587,11 +587,10 @@ type DataDeclarationArgs =
   , [DataConstructorDeclaration]
   )
 
+-- | (The infered type signatures of data constructors, The inferred kind of the declaration)
 type DataDeclarationResult =
   ( [(DataConstructorDeclaration, SourceType)]
-  -- ^ The infered type signatures of data constructors
```

Comment:
If this is causing an error, I'd rather just remove the `^`. If these are not kept inline to the fields the describe I think they are much more likely to go out of date.

Reply:
I change to inline.